km_api/know_me/factories.py

```diff
@@ -20,6 +20,17 @@ def create_file():
         name='foo.txt')
 
 
+class EmergencyItemFactory(factory.django.DjangoModelFactory):
+    """
+    Factory for generating ``EmergencyItem`` instances.
+    """
+    name = factory.Sequence(lambda n: 'Profile Item {n}'.format(n=n))
```

Comment:
The format string here should be 'Emergency Item {n}' so we can distinguish it from normal profile item.s